components/application-deployers/org.wso2.carbon.application.deployer.synapse/src/main/java/org/wso2/carbon/application/deployer/synapse/SynapseAppDeployer.java

```diff
@@ -55,6 +55,9 @@
 import org.wso2.carbon.mediation.initializer.ServiceBusUtils;
```

Comment:
Correct formatting issues in all the classes. Some lines are exceeding the allowed limit.